lambda_functions/rds_init/lambda-handler.py

```diff
@@ -0,0 +1,51 @@
+import json
+import logging
+import os
+import sys
+
+import psycopg2
+from aws_lambda_powertools.utilities import parameters
+from aws_lambda_powertools.utilities.typing import LambdaContext
+
+# This lambda function is only meant to be run once during cdk initialization,
+# as post-db creation initialization script
+
+bde_host = os.environ["BDE_HOST"]
+bde_ro_user = os.environ["BDE_RO_USER"]
+# bde_ro_cred = parameters.get_secret(os.environ["BDE_RO_USER_CRED"])
+
+rds_fdw_host = os.environ["RDS_FDW_HOST"]
+rds_fdw_db = os.environ["RDS_FDW_DB"]
+
+rds_fdw_root_cred = parameters.get_secret(os.environ["RDS_FDW_ROOT"])
+
+rds_fdw_root_cred = json.loads(rds_fdw_root_cred)  # type: ignore
+rds_fdw_root_user = rds_fdw_root_cred["username"]  # type: ignore
+rds_fdw_root_pw = rds_fdw_root_cred["password"]  # type: ignore
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+try:
+    conn = psycopg2.connect(
+        host=rds_fdw_host,
+        database=rds_fdw_db,
+        user=rds_fdw_root_user,
+        password=rds_fdw_root_pw,
+    )
+
+except psycopg2.Error as e:
+    logger.error("ERROR: Unexpected error: Could not connect to Postgresql RDS: %s", rds_fdw_host)
+    logger.error(e)
+    sys.exit()
+
+logger.info("SUCCESS: Connection to POSTGRES RDS instance succeeded")
+
+
+def handler(_event: dict[str, str], _context: LambdaContext) -> None:
+    with conn.cursor() as cur:
+        with open("init.sql", "r", encoding="utf8") as bde_fdw_sql:
```

Comment:
Since we can't really change the SQL file in the Lambda without redeploying, would it make sense to put it in a `"""` string?

Reply:
Probably OK since the query is short.